demos/demos_databases_apis/microsoft/kusto/graphistry_ADX_kusto_demo.ipynb

```diff
@@ -0,0 +1,453 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c80d3a57-7c78-4bc3-bb4d-6b8cbd71f9fc",
+   "metadata": {},
+   "source": [
+    "# Tutorial: Manage graphs with Kusto Persistent Graph & Graphistry\n",
+    "\n",
+    "## Microsoft Azure Data Explorer Update\n",
+    "\n",
+    "In a significant step forward for graph analytics, **Microsoft has introduced Persistent Graph Semantics in Azure Data Explorer (ADX)**.  \n",
```

Comment:
Positioning like 'step forward'  feels more appropriate for a news piece (blogpost, PR, ...) while the tutorial should be more evergreen. Still useful to say why important/useful, but imagine reading it 1yr from now